my-react-leaflet-app/src/components/MaterialUI/DetailsDrawer.tsx

```diff
@@ -37,7 +38,7 @@ export default function DetailsDrawer({ selectedMark }: DetailsDrawerProps) {
         setState({ ...state, [anchor]: open });
       };
 
-  const docTypeAccordions = CarceralDocument.getCarceralDocumentsByType(selectedMark || []).map((docType, index) => (
```

Comment:
I didn't see how the argument to getCarceralDocumentsByType was working. It seemed like the method was meant to operate on the dataset instead of a single point, so I switched it to take no args since the service has the full dataset. Let me know if this is actually supposed to work differently. @vitaovich

Reply:
You are right it did operate on the dataset, and is mainly used to help show the UI flow right now. 

In issue #13 I'm implementing the details user flow.

It should operate by (let me know if you got other ideas)
1. user selects a marker
2. get documents from dataset associated to that marker
3. apply any filtering based on current filter options selected (could happen together as part of step 2)
4. categorize by document type
5. get document information like title and file location